docs/manual-CN/1.overview/2.quick-start/2.FAQ.md

```diff
@@ -161,6 +162,19 @@ nebula> SHOW CONFIGS;
 
 [[↑] 回到顶部](#常见问题)
 
+### 修改  RocksDB block cache
+
+更改 storage 的配置文件 `storaged.conf`（ 路径为 `/home/user/nebula/build/install/etc`，路径可能因人而异）并重启，例如：
+
+```bash
+# Change rocksdb_block_cache to 1024
+--rocksdb_block_cache = 1024
+```
+
+使用 `curl 127.0.0.1:12114/get_flags | grep rocksdb_block_cache` 查看配置是否生效。
```

Comment:
It's better to display the result if it works

where to execute the command, on command line?

Reply:
right